docs/crds/hostrule.md

```diff
@@ -26,6 +26,7 @@ A sample HostRule CRD looks like this:
           termination: edge
         gslb:
           fqdn: foo.com
+          includeAliases: false
```

Comment:
Please add some information about this field under the section configure-gslb-fqdn in the same document( [docs/crds/hostrule.md#configure-gslb-fqdn](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/master/docs/crds/hostrule.md#configure-gslb-fqdn))

Reply:
The information has been added now.